instrumentation/graphql/lib/opentelemetry/instrumentation/graphql/tracers/graphql_trace.rb

```diff
@@ -155,7 +155,7 @@ def trace_field?(field)
             end
           end
 
-          def platform_field_key(field)
+          def opentelemetry_field_key(field)
```

Comment:
I believe what we have settled on is prefixing private methods in patches or mixins with `_otel` so I will request that you use the same naming pattern, e.g. 

```suggestion
          def _otel_field_key(field)
```

Reply:
👍 done